DOC: Add guide links to relevant API docs (#62650)
pandas-dev/pandas: doc/source/reference/series.rst

======
.. currentmodule:: pandas

.. note::

>    For more information on using Series, see the :ref:`Essential basic functionality <basics>` section of the User Guide.

Reviewer: ```suggestion
   For an introduction to using Series, see :ref:`basics`.
```
